cmake/python_qt_binding-extras.cmake

```diff
@@ -0,0 +1,2 @@
+# location of cmake files in installspace
+set(python_qt_binding_EXTRAS_DIR "${python_qt_binding_DIR}")
```

Comment:
What is this variable needed for? `python_qt_binding_DIR` should be sufficient.

Reply:
It was in the original python_qt_binding-extras.cmake.em. To port, I removed the develspace part and left the install space. Should the file be removed?

Reply:
If the variable isn't used anywhere (in downstream packages?) then it can be removed.